cypress/integration/zones.spec.ts

```diff
@@ -50,3 +50,62 @@ describe('Test zones', () => {
     cy.get('h1').should('contain.text', 'Welcome to the APPUiO Cloud Portal');
   });
 });
+
+describe('Test single zone', () => {
+  beforeEach(() => {
+    cy.setupAuth();
+    window.localStorage.setItem('hideFirstTimeLoginDialog', 'true');
+  });
+  beforeEach(() => {
+    // needed for initial getUser request
+    cy.setPermission({ verb: 'list', resource: 'zones', group: 'rbac.appuio.io' });
+    cy.intercept('GET', 'appuio-api/apis/appuio.io/v1/users/mig', {
+      body: createUser({ username: 'mig', defaultOrganizationRef: 'nxt' }),
+    });
+  });
+
+  it('displays zone details', () => {
+    cy.setPermission({ verb: 'list', resource: 'zones', group: 'appuio.io' });
+    cy.visit('/zones/cloudscale-lpg-2');
+    cy.intercept('GET', 'appuio-api/apis/appuio.io/v1/zones', {
+      body: createZoneList({ items: [zoneCloudscale1, zoneCloudscale2] }),
+    });
+    cy.get('#zone-details-title').should('contain.text', 'Zone Details');
+    cy.get('[data-cy=zone-name]').eq(0).should('contain.text', 'cloudscale.ch LPG 2');
+  });
+
+  it('displays zone details of zone with non-url conform name', () => {
```

Comment:
This case should not come up in practice. Object identifiers (as the zone name) have to follow the rules explained at https://kubernetes.io/docs/concepts/overview/working-with-objects/names/. It probably does not hurt to leave it in.